utils/build-script-impl

```diff
@@ -1540,9 +1540,14 @@ for host in "${ALL_HOSTS[@]}"; do
 
             llvm)
                 if [ "${BUILD_LLVM}" == "0" ] ; then
-                    build_targets=(clean)
+                    build_targets=()
                 fi
-                if [[ "${SKIP_BUILD}" || "${SKIP_BUILD_LLVM}" ]] ; then
+
+                if [[ "${BUILD_SWIFT_TOOLS}" == "0" && "${SWIFT_INCLUDE_TESTS}" == "0" ]] ; then
```

Comment:
It would be convenient if the second check was `("${SWIFT_INCLUDE_TESTS}" == "0" || $(is_cross_tools_host ${host}))`, as the tests aren't included when cross-compiling the standalone stdlib too.